src/domain/context/context.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { Field, InputType } from '@nestjs/graphql';
+import { MaxLength } from 'class-validator';
+import { ReferenceInput } from '../reference/reference.dto';
+
+@InputType()
+export class ContextInput {
```

Comment:
It might be worth naming all inputs Dtos (E.g. ContextDto). It's the recommended approach with nestjs and anyway we are dropping the 'nested' inputs.

Reply:
Still need this object for updates